### ISSUE TITLE
Add Homebrew formula

### ISSUE THREAD

**Issue by bbkane** [cosmtrek/air#571]
Hello - could you add a Homebrew formula for `air` so I can install and upgrade it via Homebrew. It looks like you're already using GoReleaser, and GoReleaser has a built-in  [Homebrew formula](https://goreleaser.com/customization/homebrew/) option. I use GoReleaser to [generate Homebrew formulas for my CLIs](https://github.com/bbkane/grabbit/blob/master/.goreleaser.yml) so I can attest it works well.

**Comment by bbkane** [comment]
@xiantang , I'm not sure this is a good first issue because, even though it's easy, it requires a new Homebrew formulas repo to be made and permissions to be set up so GoReleaser can push to the new repo.

New contributors won't have the permissions to do this

**Comment by xiantang** [comment]
sounds good

**Comment by vojtechmares** [comment]
Hi everyone,

I would like to add my +1 to this issue. As well as offer help with it and I am willing to maintain the Homebrew tap.

This requires the repository `air-verse/homebrew-<something>` to be created.

**Comment by xiantang** [comment]
lets work on this

**Comment by alesker** [comment]
Maybe it will be easier to add a formula to Homebrew Core and let Autobump take care of updating formula's version?

**Comment by xiantang** [comment]
yes let me prep pr for homebrew

**Comment by xiantang** [comment]
@alesker Because the air name has been occupied by another repository, I think we can do a tap first and configure the automation on our Goreleaser.

**Comment by xiantang** [comment]
If we want to add a formula, we should think of a name for air because there is another "air" in the formula.
https://formulae.brew.sh/formula/air#default

**Comment by alesker** [comment]
yeah, stumbling on that R formatter in Homebrew is how I found this issue here :)

IMO the formula name `go-air` would be pretty `brew search`-friendly, and AFAIK some other tools do use `go-` prefix even when it's not part of the project's name.

also, acknowledging the name conflict is something that we would need to do anyway to be a good brew-citizen (by adding `conflicts_with "air", because: "both install binaries with the same name"` to formula/cask).

**Comment by alesker** [comment]
anyway, here are the [changes on my homebrew-core fork](https://github.com/Homebrew/homebrew-core/compare/main...alesker:homebrew-core:go-air-formula).

if the formula looks OK to you, folks, I'll submit a PR to homebrew's base repo.

**Comment by xiantang** [comment]
> anyway, here are the [changes on my homebrew-core fork](https://github.com/Homebrew/homebrew-core/compare/main...alesker:homebrew-core:go-air-formula).
> 
> if the formula looks OK to you, folks, I'll submit a PR to homebrew's base repo.

its really good to me, please raise a pr. The name call air-go is also ok to me

**Comment by alesker** [comment]
took a bit of time, but the formula is finally merged, and [available](https://formulae.brew.sh/formula/go-air)

**Comment by alesker** [comment]
also, [added](https://github.com/air-verse/air/pull/879) Homebrew option to the Installation section in README

**Comment by xiantang** [comment]
Close many thanks @alesker
